owncloudData/src/main/java/com/owncloud/android/data/files/db/FileDao.kt

```diff
@@ -279,6 +280,7 @@ interface FileDao {
                 permissions = null,
                 treeEtag = "",
                 availableOfflineStatus = NOT_AVAILABLE_OFFLINE.ordinal,
+                lastUsage = 221
```

Comment:
Why put that time when adding a new file to de DB. Add the coma to the last parameter.

Reply:
Changed